lib/pages/athlete/AthletePage.dart

```diff
@@ -987,29 +987,34 @@ class _AthletePageState extends State<AthletePage> {
                       ),
                     ],
                   ),
-                  SizedBox(width: 30),
-                  Container(
-                    width: 100,
+                  //SizedBox(width: 0),
+                  Expanded(
```

Comment:
This will take up the remaining spacing causing the button to be stretched out. We want the size to stay consistent regardless of the screen size